bin/gpu-trace.py

```diff
@@ -535,13 +535,16 @@ def __init__(self, args):
         self.args = args
         self.server = None
         self.gpuTrace = GpuTrace()
+        self.cpuTrace = None
```

Comment:
GpuTrace() already captures some cpu trace information via ftrace. Which makes it a bit of a misnomer.

The problem is accentuated by adding a `self.cpuTrace`. What do you think of renaming this to `self.perfTrace`? Of course perf is also a loaded term so it can be confusing too.

I'm on the fence about either option so I'll go with whatever you think best.

Reply:
I agree with you that CpuTrace is a bit of an awkward name, although I had trouble coming up with a better one.  I wanted to make it clear that the "CpuTrace" class provides similar semantics to the "GpuTrace" class, although I suppose a name like "PerfTrace" would achieve the same thing.

The more I think about it, the less I like "CpuTrace" in any case, so since you're leaving it up to me, I'll rename it to "PerfTrace."